src/compiler/diagnosticMessages.json

```diff
@@ -4449,6 +4449,10 @@
         "category": "Error",
         "code": 6234
     },
+    "Disable loading projects this project references.": {

```

Comment:
I would phrase as `Disable loading referenced projects` or similar. As-is it at least needs to be `this project's references`

Reply:
`disableReferencedProjectLoad` is good